test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -0,0 +1,207 @@
+                                runas=OPER_USER)
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15007) returned from server' in e.msg[1])
+
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="testDeleteSched",
+                                runas=ROOT_USER)
+        except PbsManagerError, e:
+            self.fail(e.msg)
+
+        # Check for attribute set permission
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="TestCommonSched",
+                                runas=OPER_USER)
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15007) returned from server' in e.msg[1])
+
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="TestCommonSched",
+                                runas=ROOT_USER)
+        except PbsManagerError, e:
+            self.fail(e.msg)
+
+    def test_delete_default_sched(self):
+        """
+        Delete default scheduler.
+        """
+        try:
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="default")
+        except PbsManagerError, e:
+            if self.server.get_op_mode() == PTL_CLI:
+                self.assertTrue(
+                    'qmgr: Error (15214) returned from server' in e.msg[1])
+
+    def test_set_sched_attributes(self):
+        """
+        Set an attribute of a scheduler object .
+        """
+        try:
+            # Default values will be set
+            self.server.manager(MGR_CMD_SET,
+                                SCHED,
+                                {'sched_cycle_length': 1234},
+                                id="TestCommonSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
+        # rc = self.server.manager(MGR_CMD_LIST, SCHED, id="TestCommonSched")
+        # TODO call the list using qmgr and assert the attribute
+
+    def test_unset_sched_attributes(self):
+        """
+        Unset a attribute of a scheduler object .
+        """
+        try:
+            self.server.manager(MGR_CMD_UNSET,
+                                SCHED,
+                                'sched_cycle_length',
+                                id="TestCommonSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
+        # rc = self.server.manager(MGR_CMD_LIST, SCHED, id="TestCommonSched")
+        # TODO call the list using qmgr and assert the default attribute value
```

Comment:
Actually if you're just doing a list to make sure your set/unset operations happened, the manager() expect=True parameter does that.  It first does the set/unset and then queries to make sure it happened.  I highly suspect that it is broken now if the list scheduler functionality is not working.